superdesk/publish/subscriber_token.py

```diff
@@ -4,6 +4,7 @@
 from datetime import timedelta
 from superdesk.utc import utcnow
 from superdesk.utils import get_random_token
+from content_api import MONGO_PREFIX
```

Comment:
I think we could keep this for now.

But need to think about that superdesk shouldn't use imports from `conten_api` and `apps` I don't know how it could be done yet... I also think that `legal_archive` and `archived` also have these prefixes so maybe it should also exist an ability to run archive as separate WSGI application, etc. Just need to look over all this code and think.